### PR TITLE
add missing supported chains

### PR DESCRIPTION
**Pull request by r1ck12**
Added Supported Chains: 
* Algorand Archival
* Algorand Mainnet
* AVAX Archival
* AVAX_CRA
* AVAX_DFK
* Binance Smart Chain
* Binance Smart Chain (Archival)
* Ethereum Mainnet High Gas
* Kava Mainnet
* Kava Mainnet Archival
* Oasys Mainnet
* Oasys Mainnet Archival
* OKC Mainnet
* Polygon zkEVM Mainnet 
* Ropsten
* Scroll testnet
* Sui mainnet

Updated: 
* Ethereum Mainnet Archival with trace calls (fixed Portal API prefix)
* Gnosis Chain Archival
* Gnosis Chain Mainnet

### CODE REVIEW

**Review by ArtSabintsev — APPROVED** [review]

**Comment by ArtSabintsev** [comment]
@r1ck12 can you add Sui (0076). I think it's sui-mainnet in portal.

**Review by ArtSabintsev — CHANGES_REQUESTED** [review]
Left feedback in a comment re: Sui

File: content/supported-blockchains/_index.md [hunk 1]
```diff
@@ -16,47 +16,62 @@ Pocket can be made to work with any network/blockchain (RelayChain) that uses th
...
+| Pocket Network Mainnet                      | mainnet                   | 0001 
+| Polygon Matic                               | poly-mainnet              | 0009 
+| Polygon Matic Archival                      | poly-archival             | 000B
+| Polygon Mumbai                              | polygon-mumbai            | 000F 
+| Ropsten                                     | eth-ropsten               | 0023 
+| Scroll testnet                              | scroll-testnet            | 0075 
+| Solana                                      | solana-mainnet            | 0006 
+| Starknet Mainnet                            | starknet-mainnet          | 0060
+| Starknet Testnet                            | starknet-testnet          | 0061
+| Sui mainnet                                 | sui-mainnet               | 0076
```
[the remark this answers is not in the document]
Comment (r1ck12) [reply content/supported-blockchains/_index.md]: @ArtSabintsev fixed

File: content/supported-blockchains/_index.md [hunk 2]
```diff
@@ -16,47 +16,62 @@ Pocket can be made to work with any network/blockchain (RelayChain) that uses th
...
+| Metis Mainnet                               | metis-mainnet             | 0058 
+| Moonbeam Mainnet                            | moonbeam-mainnet          | 0050
+| Moonriver Mainnet                           | moonriver-mainnet         | 0051
+| NEAR Mainnet                                | near-mainnet              | 0052 
+| OKC Mainnet                                 | oKc-mainnet               | 0047 
+| Oasys Mainnet                               | oasys-mainnet             | 0070 
+| Oasys Mainnet Archival                      | oasys-mainnet-archival    | 0069
+| Optimism Mainnet                            | optimism-mainnet          | 0053
+| Osmosis Mainnet                             | osmosis-mainnet           | 0054 
+| POLYGON ZKEVM Mainnet                       | polygon-zkevm-mainnet     | 0074
```
[the remark this answers is not in the document]
Comment (ArtSabintsev) [reply content/supported-blockchains/_index.md:61]: @r1ck12 following up here. Happy to merge after, expecially if you add swimmer and DFK back.

File: content/supported-blockchains/_index.md [hunk 3]
```diff
@@ -16,47 +16,62 @@ Pocket can be made to work with any network/blockchain (RelayChain) that uses th
...
+| Fantom Mainnet                              | fantom-mainnet            | 0049 
+| Fuse                                        | fuse-mainnet              | 0005 
+| Fuse Archival                               | fuse-archival             | 000A 
+| Gnosis Chain Archival                       | poa-xdai-archival         | 000C
+| Gnosis Chain Mainnet                        | poa-xdai                  | 0027 
+| Goerli                                      | eth-goerli                | 0026 
+| Goerli Archival                             | goerli-archival           | 0063 
+| Harmony Shard 0                             | harmony-0                 | 0040 
+| IoTeX Mainnet                               | iotex-mainnet             | 0044 
+| KAVA Mainnet Archival                       | kava-mainnet-archival     | 0072
```
Comment (ArtSabintsev) [inline content/supported-blockchains/_index.md:47]: Capital-case for Kava, and put the Archival _below_ the mainnet one.

File: content/supported-blockchains/_index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (r1ck12) [reply content/supported-blockchains/_index.md]: done

File: content/supported-blockchains/_index.md (hunk 3 above)
Comment (r1ck12) [reply content/supported-blockchains/_index.md]: done 👍

**Review by ArtSabintsev — APPROVED** [review]

**Pull request merged**